app/src/main/cpp/skyline/services/account/IProfile.h

```diff
@@ -18,6 +18,18 @@ namespace skyline::service::account {
       private:
         UserId userId;
 
+        static constexpr std::array<u8, 107> profileImageIcon{
```

Comment:
What is this

Reply:
I would also just store it in the cpp file itself

Reply:
> What is this

image, in bytes array

Reply:
I know that lol but what's the image

Reply:
> I know that lol but what's the image

the smallest image I could find https://github.com/mathiasbynens/small/blob/master/jpeg.jpg

Reply:
Ah, move this to the cpp file and comment as such then